app/views/submissions/_version_dropdown.html.erb

```diff
@@ -1,10 +1,14 @@
 <span id="version-dropdown">
   <strong>version:</strong>
-  <select class="browser-default" onchange="location.assign(this.value)" title="* version also contains the currently selected file">
+  <select
+    class="browser-default"
+    onchange="location.assign(this.value)"
```

Comment:
There's a warning here about ["DOM text reinterpreted as HTML"](https://github.com/autolab/Autolab/security/code-scanning/38) in case it doesn't show up for you!

Reply:
seems to be a false positive: https://github.com/autolab/Autolab/security/code-scanning/38

Reply:
Yes, it's a false positive which I couldn't get to disappear